packages/client-engine-runtime/src/transactionManager/TransactionManager.ts

```diff
@@ -39,13 +40,27 @@ export class TransactionManager {
   private closedTransactions: TransactionWrapper[] = []
   private readonly driverAdapter: SqlDriverAdapter
   private readonly transactionOptions: Options
-
-  constructor({ driverAdapter, transactionOptions }: { driverAdapter: SqlDriverAdapter; transactionOptions: Options }) {
+  private readonly tracingHelper: TracingHelper
+
+  constructor({
```

Comment:
We also have `new TransactionManager` in `worker.ts` in the `prisma-engines` repo. We need a PR on the `prisma-engines` side, so the tests will pass there once this PR is on `main`.

Reply:
Good call out, done in https://github.com/prisma/prisma-engines/pull/5329